packages/json-schema/src/lib.ts

```diff
@@ -0,0 +1,50 @@
+import { createTypeSpecLibrary, JSONSchemaType } from "@typespec/compiler";
+
+export type FileType = "yaml" | "json";
+export type Int64Strategy = "string" | "number";
+
+export interface JSONSchemaEmitterOptions {
+  /**
+   * If the content should be serialized as YAML or JSON.
+   * @default yaml, it not specified infer from the `output-file` extension
+   */
+  "file-type"?: FileType;
+
+  /**
+   * How to handle 64 bit integers on the wire. Options are:
+   *
+   * * string: serialize as a string (widely interoperable)
+   * * number: serialize as a number (not widely interoperable)
+   */
+  "int64-strategy"?: Int64Strategy;
+
+  /**
+   * When true, bundle all the schemas into a single json schema document
+   * with schemas under $defs.
+   */
+  bundle?: boolean;
+}
+
+const EmitterOptionsSchema: JSONSchemaType<JSONSchemaEmitterOptions> = {
+  type: "object",
+  additionalProperties: false,
+  properties: {
+    "file-type": { type: "string", enum: ["yaml", "json"], nullable: true },
```

Comment:
I would copy the description here too so it shows up in the ref doc, which also make me think that we might want this to show up in the website no?

Reply:
It should show up on the website. As for manually written docs, I'll do a fast follow with those!